frontend/src/utils/crypto.ts

```diff
@@ -0,0 +1,67 @@
+import { Chain } from '../model/chain';
+
+export function convertChainSymbolToChain(symbol: string) {
+  switch (symbol) {
+    case 'ETH':
+      return Chain.Ethereum;
+    case 'BTC':
+      return Chain.Bitcoin;
+    case 'BCH':
+      return Chain.BitcoinCash;
+    case 'LTC':
+      return Chain.Litecoin;
+    case 'BNB':
+      return 'BNB';
+    case 'AVAX':
+      return Chain.Avalanche;
+    case 'THOR':
+      return Chain.THORChain;
+    case 'GAIA':
+      return Chain.Cosmos;
+    case 'KUJI':
+      return Chain.Kujira;
+    case 'ARB':
+      return Chain.Arbitrum;
+    case 'MAYA':
+      return Chain.MayaChain;
+    case 'DOGE':
+      return Chain.Dogecoin;
+    case 'DASH':
+      return Chain.Dash;
+    case 'BSC':
+      return Chain.BSC;
+    default:
+      return symbol;
+  }
+}
+
+export function convertChainToChainTicker(chain: Chain) {
```

Comment:
I think it's the same as calling `getChainFeeCoin(chain).ticker`

Reply:
Not the same. For example, for THORChain example you provided returns RUNE instead of THOR, for MAYAChain it returns CACAO instead of MAYA etc.